src/pybind/mgr/rook/rook_cluster.py

```diff
@@ -335,6 +335,8 @@ def predicate(item):
                 "labels": d['metadata']['labels'],
                 'phase': d['status']['phase'],
                 'container_image_name': image_name,
+                'container_image_id': d['status']['container_statuses'][0]['image_id'].split('@sha256:')[1],
```

Comment:
It seems awkward to extract these details. Are the `@sha256` and `docker://` 100% reliable in all types of K8s clusters? Maybe they are, I'm just not sure. Even better if we don't actually need to use them anywhere, from the question above.

Reply:
They are not. I have added additional check for `sha256` and removed `docker://` as it is not reliable.